pkgs/desktops/gnome-3/core/mutter/default.nix

```diff
@@ -30,15 +30,15 @@ stdenv.mkDerivation rec {
   ];
 
   propagatedBuildInputs = [
-    # required for pkgconfig to detect mutter-clutter
-    libXtst
+    libXtst # required for pkgconfig to detect mutter-clutter
+    clutter
```

Comment:
@jtojnar We're sure this is needed right?

Reply:
Actually, I do not see the clutter requirement. `libmutter-2` requires `mutter-clutter-2` and `x11`. `mutter-clutter-2` requires `mutter-cogl-2` and `xtst` (among others), but `rg clutter $(nix-build -A gnome3.mutter)/lib/pkgconfig/` leaves me empty handed.

Reply:
I recall that I did this because of something weird with https://github.com/elementary/gala/blob/master/meson.build

Reply:
I would try dropping 76fa11b and adding clutter directly to gala. From nix point of view, it should be the same.

Reply:
I think I know what is going on. `mutter-clutter` does not depend on `clutter` but depends on much the same things clutter propagates. I would just keep the status quo for now.

Reply:
Ahh that makes sense. But currently this doesn't seem right.